src/datetimepicker.ios.js

```diff
@@ -109,7 +110,7 @@ export default function Picker({
   return (
     <RNDateTimePicker
       testID={testID}
-      ref={_picker}
+      ref={forwardRef || _picker}
```

Comment:
hello, thanks for the PR. This is not going to work because we need to use the ref in https://github.com/react-native-datetimepicker/datetimepicker/blob/4680031c15ac39288c9efe3900d7a6c473d68ef8/src/datetimepicker.ios.js#L64

Reply:
For the next person that has a go at this, you can fork the ref and have both, using something like [apply-ref](https://github.com/mitchellhamilton/apply-ref).